functions/Connect-DbaInstance.ps1

```diff
@@ -300,7 +300,8 @@ function Connect-DbaInstance {
         [string]$Thumbprint = (Get-DbatoolsConfigValue -FullName 'azure.certificate.thumbprint'),
         [ValidateSet('CurrentUser', 'LocalMachine')]
         [string]$Store = (Get-DbatoolsConfigValue -FullName 'azure.certificate.store'),
-        [switch]$DisableException
+        [switch]$DisableException,
+        [switch]$VeryVerbose
```

Comment:
We have support for this within our message system.

You can add this value as a message type to the Write-Message call instead of using a parameter on the command itself.

Reply:
Ok, good to know. Can you give me a hint where to find more info or an example?
And someone should rephrase some of the messages if this should ever go into production.
This is only a first step to give more insight about how and where the values are set.